pages/instances/troubleshooting/fix-common-issues.mdx

```diff
@@ -100,6 +101,16 @@ This page lists the most common issues you may encounter with your Scaleway Inst
 - Re-add your SSH Key in the Scaleway console:
    - Go to the [SSH keys](https://console.scaleway.com/project/ssh-keys) tab in the **Project** section of theScaleway console, remove the old SSH key, then add your new, correct public key.
    - Restart your Instance to re-upload the list of allowed SSH keys.
+- Confirm that your instance has access to the api-metadata by using the serial console
+   - Execute the following :
+     ```bash
+     # curl http://169.254.42.42/
+     ```
+     The response should be
+     ```json
+     {"api": "api-metadata", "description": "Metadata API, just query http://169.254.42.42/conf or http://169.254.42.42/conf?format=json to get info about yourself", "version": "12.23.0"}
+     ```
+     Otherwise you need to identify why the access to api-metadata is blocked somehow.
```

Comment:
```suggestion
- Make sure that your Instance has access to the api-metadata by running the following command in the serial console:
     ```bash
     # curl http://169.254.42.42/
     ```
     The following output should display:
     ```json
     {"api": "api-metadata", "description": "Metadata API, just query http://169.254.42.42/conf or http://169.254.42.42/conf?format=json to get info about yourself", "version": "12.23.0"}
     ```
     No output means that access to the api-metadata is blocked. Identify the reason for the block and try running the command above again.
```